.nvmrc

```diff
@@ -0,0 +1 @@
+22.11.0
```

Comment:
Does this mean you are planning to self-host from the beginning? You mentioned [here](https://github.com/MakePrisms/boardwalkcash/pull/119#issuecomment-2446317431) that max node version of vercel is 20

Reply:
that was the case back then but they updated it recently so they support 22 too now